packages/web/components/chart/token-pair-historical.tsx

```diff
@@ -80,15 +90,40 @@ const TokenPairHistoricalChart: FunctionComponent<{
           <AnimatedAxis orientation="bottom" numTicks={4} />
           <AnimatedAxis orientation="left" numTicks={5} strokeWidth={0} />
           <AnimatedGrid columns={false} numTicks={5} />
-          <AnimatedLineSeries
-            key={data.length}
-            dataKey="close"
-            data={data}
-            curve={curveNatural}
-            xAccessor={(d: { time: number; close: number }) => d?.time}
-            yAccessor={(d: { time: number; close: number }) => d?.close}
-            stroke={theme.colors.wosmongton["200"]}
-          />
+
+          {showGradient ? (
+            <>
+              <AnimatedAreaSeries
+                dataKey="close"
+                data={data}
+                xAccessor={(d: { time: number; close: number }) => d?.time}
+                yAccessor={(d: { time: number; close: number }) => d?.close}
+                fillOpacity={0.4}
+                curve={curveNatural}
+                fill="url(#gradient)"
+              />
+              <LinearGradient
+                id="gradient"
+                from="#C41BFF"
```

Comment:
can these be tailwind variables?

Reply:
FYI @JoseRFelix it's possible to import the tailwind config

Reply:
Thanks for the suggestion! Added the variables on my new commit